torch/csrc/jit/mobile/model_compatibility.h

```diff
@@ -20,5 +21,20 @@ TORCH_API int64_t _get_model_bytecode_version(const std::string& filename);
 TORCH_API int64_t _get_model_bytecode_version(
     std::shared_ptr<caffe2::serialize::ReadAdapterInterface> rai);
 
+// Struct storing metadata of an operator that can be useful for versioning
+struct OperatorInfo {
```

Comment:
Do you envision more stuff being added here?

Reply:
Yes. At the very least backend info will be useful from the runtimes side. Im planning on sharing this struct between runtime and model. In the future we might move to a full operator versioning system and then that would go inside this struct too.